src/TodoListItem.js

```diff
@@ -1,8 +1,11 @@
 import React from "react";
 
-function TodoListItem({ todoTitle }) { 
+function TodoListItem({ todoTitle, onRemoveTodo }) { 
```

Comment:
So here you would pass in the props like this

```suggestion
function TodoListItem({ todo, onRemoveTodo }) { 
```